javatools/src/main/java/org/xvm/asm/constants/ParameterizedTypeConstant.java

```diff
@@ -814,6 +808,34 @@ public ResolutionResult resolveContributedName(
         return result;
         }
 
+    @Override
+    public TypeConstant widenEnumValueTypes()
+        {
+        TypeConstant   constUnderlying = m_constType;
+        boolean        fDiff           = false;
```

Comment:
`fDiff` will always be the same as `(aconstOriginal != aconstResolved)`

Reply:
Changed accordingly